packages/material-ui/src/styles/colorManipulator.js

```diff
@@ -31,7 +31,15 @@ function clamp(value, min = 0, max = 1) {
  * @param {string} color - Hex color, i.e. #nnn or #nnnnnn
  * @returns {string} A CSS rgb color string
  */
-export function convertHexToRGB(color) {
+export function hexToRgb(color) {
```

Comment:
rename for consistency cc @mbrookes

Reply:
I already did the same in my color-manipulator-naming branch (PR was waiting to agree the naming for other functions https://github.com/mui-org/material-ui/issues/13039#issuecomment-476020214), so: 👍

Reply:
I was planning on updating this pull request to match @eps1lon review. I think that we can go with the fallback strategy + warning.

Reply:
Ok, I'm finishing the effort so you can rebase :).